pallets/loans/src/interest.rs

```diff
@@ -0,0 +1,119 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use primitives::{Balance, CurrencyId};
+use sp_runtime::{traits::Zero, DispatchResult};
+
+use crate::*;
+
+impl<T: Config> Pallet<T> {
+    /// Accrue interest per block and update corresponding storage
+    pub(crate) fn accrue_interest() -> DispatchResult {
```

Comment:
is `pub(crate)` necessary? is it used in test? I think maybe `fn` is already enough

Reply:
Yes, otherwise we can not invoke it in lib.rs and test.rs.